modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc

```diff
@@ -370,129 +362,132 @@ If you want to update several processes, simply run the command with all the pro
-* execute the SQL requests to GRANT the proper rights to the Oracle users; for both Bonita BPM and BDM schemas
+. Configure the Oracle database server, in particular activate the XA transactions management: see the _Oracle Database_ section in the link:database-configuration[Database creation and configuration for Bonita engine and BDM] page
+. Install the missing Oracle components
+. Execute the SQL scripts to _install_ XA management elements
+. Execute the SQL requests to GRANT the proper rights to the Oracle users; for both Bonita BPM and BDM schemas
 
-=== Download the specific jdbc driver for the Oracle 12c (12.2.0.x.y) or 19c (19.3.0.0)
+==== Download the specific jdbc driver for the Oracle 12c (12.2.0.x.y) or 19c (19.3.0.0)
 
-*Beware*: two different jdbc driver jar files may share the same name (ojdbc8.jar).
+*Caution*: two different jdbc driver jar files may share the same name (ojdbc8.jar). +
 
-Each file however is specific to the Oracle DB server version installed.
+Each file however is specific to the Oracle DB server version installed. +
 Please make sure to download the appropriate one:
 
 * Oracle 12c (12.2.0.x.y) : Driver ojdbc8.jar https://www.oracle.com/database/technologies/jdbc-ucp-122-downloads.html[Oracle Database 12.2.0.1 JDBC Driver & UCP Downloads] ( make sure it is the official driver by checking the SHA1 Checksum: 60f439fd01536508df32658d0a416c49ac6f07fb )
 * Oracle 19c (19.3.0.0) : Driver ojdbc8.jar https://www.oracle.com/database/technologies/appdev/jdbc-ucp-19c-downloads.html[Oracle Database 19c (19.3) JDBC Driver & UCP Downloads] ( make sure it is the official driver by checking the SHA1 Checksum: 967c0b1a2d5b1435324de34a9b8018d294f8f47b )
 
-*Note I*: The migration tool already includes the oracle driver for Oracle 19c (19.3.0.0) in the `bonita-migration/lib` directory. If your are not using Oracle 19c (19.3.0.0) you need to replace it.
+*Note*: Bonita Migration Tool already includes the Oracle driver for Oracle 19c (19.3.0.0) in the `bonita-migration/lib` directory. If your are not using Oracle 19c (19.3.0.0) you need to replace it.
 
-=== Check the Bonita 7.8.4 server starts with the Oracle database server 12c (12.2.0.x.y) or 19c (19.3.0.0)
+==== Check the Bonita 7.8.4 server starts with the Oracle database server 12c (12.2.0.x.y) or 19c (19.3.0.0)
 
-* download and install a Bonita 7.8.4 server
-* setup the Bonita 7.8.4 server to use the Oracle 12c (12.2.0.x.y) or 19c (19.3.0.0) database
-* request and install a temporary 7.8 license in the Bonita server
-* start the Bonita 7.8.4 server
-* check you can successfully log into the portal
+. Download and install a Bonita 7.8.4 bundle
+. Setup the Bonita 7.8.4 bundle to use the Oracle 12c (12.2.0.x.y) or 19c (19.3.0.0) database
+. Request and install a temporary 7.8 license in the Bonita bundle
+. Start the Bonita 7.8.4 bundle
+. Check that you can successfully log into the Bonita user interfaces: Portal (before Bonita 2021.2) or applications.
```

Comment:
Same comment again, remove the before 2021.2 part.

Reply:
Some customers will migrate to Bonita versions that are older than 2021.2. This is ok for some time, until we do not support Bonita versions older than 2021.2. 
We need to cover such scenarios in the documentation.

Reply:
removed